tests/integration_tests/functional/test_shut_down.py

```diff
@@ -57,7 +57,12 @@ def test_reboot(test_microvm_with_api):
     assert len(datapoints) == 2
 
     if platform.machine() != "x86_64":
-        vm.check_log_message("Received KVM_SYSTEM_EVENT: type: 2, event: 0")
+        message = (
+            "Received KVM_SYSTEM_EVENT: type: 2, event: [0]"
+            if "6.1" in platform.release()
```

Comment:
same as with 1.6 backport. We should cherry-pick this one: 91188f36a1e517beb0429a0f4811228e256dc814 as well